openml/extensions/sklearn/extension.py

```diff
@@ -1029,11 +1031,15 @@ def _run_model_on_fold(
         self,
         model: Any,
         task: 'OpenMLTask',
+        X_train: Union[np.ndarray, scipy.sparse.spmatrix, pd.DataFrame],
+        y_train: np.ndarray,
         rep_no: int,
         fold_no: int,
         sample_no: int,
         add_local_measures: bool,
```

Comment:
can be removed as it can be handled a level up

Reply:
Done